ibis/bigquery/tests/conftest.py

```diff
@@ -0,0 +1,80 @@
+import uuid
+import pytest
+
```

Comment:
need a ``bq = pytest.importorskip('google.datalab.bigquery')``

Reply:
addressed in [TST: use pytest.importorskip for bigquery dependency](https://github.com/ibis-project/ibis/pull/1170/commits/65ae6d84e853710e2fef03bf52470f89a422f6c8)